### PR TITLE
[JBIDE-20974] Fix project combo regression in deploy image

### PR DESCRIPTION
**Pull request by jcantrill**
* Fixes regression in populating the project combo
* An issue using ConnectionAware with BeanProperties where setConnction wasnt found
* The project combo height when there are initially no entries

### CODE REVIEW

**Comment by jcantrill** [comment]
@adietish @fbricon  please review

File: plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardModel.java [hunk 1]
```diff
@@ -37,9 +37,8 @@ public IConnection getConnection() {
 	}
 
 	@Override
-	public IConnection setConnection(IConnection connection) {
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/connection/ConnectionWizardModel.java]: how is this connected to the erroneous behaviour?

**Comment by adietish** [comment]
+1 LGTM

**Comment by jcantrill** [comment]
Closing.  Pushed to 4.3.x and cherry-picked into master

**Pull request closed without merging**